### DISCUSSION TITLE
I need exploits

### DISCUSSION THREAD

**Discussion by rolexwatch123** [3kh0/ext-remover#1046] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Are there any exploits for chromeos 105 like skiovox I am not signed in rn to avoid updates

**Comment by ghost** [comment]
personally i would try to sign in without wifi (to avoid checking updates), get caub HTML on a USB, caub urself and then restart your Chromebook. then when you sign back on, you will not update to preserve this beautiful V105 Chrome OS state. If you want an exploit for this version you can use sh1mmer or cryptosmite to unenroll and then you can do whatever you want.

**Reply by rolexwatch123** [reply]
I don’t have a usb

**Reply by ghost** [reply]
does ur chromebook already have caub installed on it?

**Reply by ghost** [reply]
if it doesn't and you can't get a USB, you can use CAUB Flags Prevent Updates if you have chrome://flags unblocked.

**Reply by rolexwatch123** [reply]
it used to but I broke and it’s board was replaced and I got it back on 105 for some reason

**Reply by rolexwatch123** [reply]
That’s blocked too

**Reply by ghost** [reply]
just get a usb then. its the safest way to keep this update state

**Reply by rolexwatch123** [reply]
but is skiovox available on 105?

**Reply by ghost** [reply]
it is but i dont recommend doing it till you have caub to prevent updates. Also skiovox isn't the best exploit for this version.

**Reply by rolexwatch123** [reply]
Last time I enabled caub in skiovox and it worked

**Reply by rolexwatch123** [reply]
caub is on now

**Reply by ghost** [reply, marked as answer]
great! now you can perform almost any exploit on this repo. if you are looking for more exploits in a easy format just for ur version u can also go to [chrose.netlify.app](https://chrose.netlify.app/)
